tests/package.json

```diff
@@ -26,9 +27,10 @@
     "@react-native-firebase/remote-config": "20.1.0",
     "@react-native-firebase/storage": "20.1.0",
     "postinstall-postinstall": "2.1.0",
-    "react": "18.3.1",
-    "react-native": "0.74.2",
+    "react": "18.2.0",
+    "react-native": "0.73.4",
```

Comment:
the only bummer on macOS support - it always lags mainline a major version or so...

Reply:
Yeah we can probably make a separate test app for macos at some point

But for now stop gap solution 🙂